lib/manager/ansible-galaxy/extract.spec.ts

```diff
@@ -22,7 +22,7 @@ describe('lib/manager/ansible-galaxy/extract', () => {
     it('extracts multiple dependencies from requirements.yml', () => {
       const res = extractPackageFile(yamlFile1);
       expect(res.deps).toMatchSnapshot();
-      expect(res.deps).toHaveLength(5);
+      expect(res.deps).toHaveLength(6);
```

Comment:
Why was this one left out completely before? I would have expected it to be left in but with a `skipReason`

Reply:
That's because of `finalize()` function which is condition for adding `dep` to the result set.

Reply:
Since now we're returning true even for invalid `currentValue`s, it appeared in snapshot. There is similar situation with `swift` manager.

Reply:
Can you double check that this new approach doesn't cause any problems later in our logic? i.e. there's no logic between this and the process/lookup logic later that could be affected?

Reply:
I can see nothing in the code that can affect it's current behavior, except `lookupUpdates()` function that calls `log.trace()` with `currentValue` which can be a trash. Though I'll check it in my test repo with limited managers, this PR needs to be monitored for errors after deploy.

Reply:
OK, let me know once you have a high confidence

Reply:
UPD: My tests are okay